src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs

```diff
@@ -31,16 +31,7 @@ static class C {
             compilation.GetDiagnostics().Verify();
             compilation.GetEmitDiagnostics().Verify(
                 // warning CS8021: No value for RuntimeMetadataVersion found. No assembly containing System.Object was found nor was a value for RuntimeMetadataVersion specified through options.
-                Diagnostic(ErrorCode.WRN_NoRuntimeMetadataVersion).WithLocation(1, 1),
-                // (9,48): error CS0518: Predefined type 'System.Nullable`1' is not defined or imported
```

Comment:
We aren't hitting them because those code paths don't exist any longer. We use a different lowering strategy for this code now.